DirectEmployers/MyJobs
# [PD-3722] table page size broken

## What's this PR do?
PR fixes the page sizing that was broken on the analytics tables

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of any analytics app

## How should this be manually tested?
Pull down the repository and go into the analytics application. Job Titles might be the best dimension to test but in the table, change the page size and you should see it work now instead of throwing an error.

## Any background context you want to provide?
